docs/dev-env-first-time-contributors.md

```diff
@@ -711,6 +711,32 @@ This is equivalent of running a halt followed by an up (aka rebooting
 the guest).  After this, you can do `vagrant provision` and `vagrant
 ssh`.
 
+#### ssl read error
+
+If you receive the following error while running `vagrant up`:
+
+```
+SSL read: error:00000000:lib(0):func(0):reason(0), errno 104
+```
+
+It means that either your network connection is unstable or speed
```

Comment:
A more succinct way to convey this: "It means your internet connection is either unstable or slow. To resolve this, run `vagrant up` until it works."

Reply:
@KiranShahdadpuri why do you prefer to state it as "speed is very less"? 'less' needs another object to be compared to.